src/instana/util/traceutils.py

```diff
@@ -24,8 +24,11 @@ def get_active_tracer() -> Optional[InstanaTracer]:
     try:
         # ToDo: Might have to add additional stuff when testing with async and tornado tracer
         current_span = get_current_span()
-        if current_span and current_span.is_recording():
-            return tracer
+        if current_span:
+            if current_span.is_recording() or current_span.name == "asyncio":
```

Comment:
Can we have a comment here stating something like "asyncio is included here because it is used as a NonRecording span solely used for context propagation"?

Reply:
Done